docs/source/quickstart.rst

```diff
@@ -3,76 +3,178 @@
 
-    For more information on deployment, see :ref:`administration` and :ref:`configuration`
+Baselines are provided in config_examples/ :
 
-Start wis2box with Docker Compose and login to the wis2box container:
+* config_examples/data-mappings.yml.example-synop-bufr, input is .bufr containing SYNOP observation-data
+* config_examples/data-mappings.yml.example-synop-csv, input is .csv containing SYNOP observation-data
+
+For example for publishing .bufr files with SYNOP data: 
+Copy this file in the directory you defined for /your/data/directory/
 
 .. code-block:: bash
 
-    python3 wis2box-ctl.py start
-    python3 wis2box-ctl.py status --all # The --all flag shows all containers, even ones that are down.
-    python3 wis2box-ctl.py login
+    cp config_examples/data-mapping.yml.example-synop-bufr /your/data/directory/data-mappings.yml
+    
+Edit /your/data/directory/data-mappings.yml and change 'ISO3C_country.center_id.data.core.weather.surface-based-observations.SYNOP':
+    
+    * replace 'ISO3C_country' with your corresponding ISO 3166 alpha-3 code.
+    * replace 'center_id' with the string identifying the center running the wis2node.
 
+wis2box needs to have a station_list.csv that contains the stations you will process, an example is provided in config_example/station_list.csv.example
+Copy this file in the directory you defined for /your/data/directory/
 
-Once logged in, verify the enviroment:
+.. code-block:: bash
+
+    cp config_examples/station_list.csv.example /your/data/directory/station_list.csv
+
+And update the file for your stations.
+
+To enable the wis2box-api and wis2box-ui to show your data disovery-metadata needs to be setup. You can setup a metadata-discovery file from the example
 
 .. code-block:: bash
 
-    wis2box environment show
+    cp config_examples/surface-weather-observations.yml /your/data/directory/surface-weather-observations.yml
 
+And edit the file /your/data/directory/surface-weather-observations.yml to provide the correct metadata for your dataset:
 
-Setup observation data processing and API publication:
+* replace 'ISO3C_country.center_id.data.core.weather.surface-based-observations.SYNOP' with the topic you used in data-mappings.yml previously*
+
+* text provided in title and abstract will be displayed in wis2box-ui *
+
+* provide a valid bounding-box in bbox *
+
+wis2box build
+-------------
+
+Please run the 'build'-command when setting up wis2box for the first time. This will start the process of building the wis2box containers from source.
 
 .. code-block:: bash
+    
+    python3 wis2box-ctl.py build
 
-    wis2box api add-collection --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed $WIS2BOX_DATADIR/metadata/discovery/mw-surface-weather-observations.yml
+This might take a while. 
 
+wis2box start
+-------------
 
-Cache and publish station collection and discovery metadata to the API:
+Start wis2box with Docker Compose and login to the wis2box container:
```

Comment:
As written I would wonder how to start the box using Docker Compose (I would be looking for a docker-compose up). What about

"Start wis2box with the wis2box command line tool and log in to the wis2box container. Note, this calls docker-compose in the background."